projects/packages/search/composer.json

```diff
@@ -46,6 +47,9 @@
 			"@composer phpunit"
 		],
 		"post-update-cmd": "php -r \"copy('vendor/automattic/wordbless/src/dbless-wpdb.php', 'wordpress/wp-content/db.php');\"",
+		"skip-test-php": [
```

Comment:
Can I ask why we skip tests for lower PHP versions?

Reply:
Ah, good question: PHPUnit versions 8.5.12 and above require PHP >= 7.2.0. I went ahead and added this in the code comments.

Reply:
I'm a bit concerned if we'd possibly have some PHP compatibility issue not caught if we skip those, would we? I looked at a couple of other package/plugin, I think they just ignored PHP version requirement. If phpunit doesn't break  for lower PHP versions in our case. Is it better we still run the tests?

Reply:
It does break for PHPUnit specified in the composer file; let me investigate downgrading our PHPUnit version as much as I can before our tests begin to fail.

Reply:
Fixed!

Reply:
Fixed!